operator/controllers/kyma_controller.go

```diff
@@ -368,6 +368,10 @@ func (r *KymaReconciler) CreateOrUpdateModules(ctx context.Context, kyma *v1alph
 			return true, nil
 		}
 		update := func() (bool, error) {
+			if module.Template.Spec.Target == v1alpha1.TargetControlPlane {
```

Comment:
This logic does not need the filter I think since there should be no case in which the target is for the control-plane and the reconciliation is triggered.

Reply:
I think by the current implementation, although reconciliation by watch the template change is prevented within `template_change.go`,  The template outdated check is still happens in `HandleConsistencyChanges` and `HandleProcessingState`, so the update will be triggered in the following `if module.TemplateOutdated`. If we don't do template outdated check for control plane CR, then this update will not happen. But I think even in this case, put a check here as a defensive statement is also fine.